### PR TITLE
feat: add docs for new recommendations (DA-1498)

### PR DESCRIPTION
**Pull request by ankur-arch**

### CODE REVIEW

File: content/700-optimize/400-recommendations/400-repeated-query.mdx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+title: 'Repeated query'
+metaTitle: 'Optimize Recommendations: Repeated query'
+metaDescription: "Learn about the recommendation provided by Optimize for repeated queries."
+tocDepth: 3
+toc: true
+---
+
+The following query targeting the `post` model is executed repeatedly with identical parameters:
```
Comment (nikolasburk) [inline content/700-optimize/400-recommendations/400-repeated-query.mdx]:
```suggestion
The following query targeting the `Post` model is executed repeatedly with identical parameters:
```

File: content/700-optimize/300-recordings.mdx [hunk 2]
```diff
@@ -39,6 +39,8 @@ When a recording session ends, Optimize generates recommendations such as:
 - [Excessive number of rows returned](/optimize/recommendations/excessive-number-of-rows-returned)
 - [Full table scans caused by LIKE operations](/optimize/recommendations/full-table-scans-caused-by-like-operations)
 - [Queries on unindexed columns](/optimize/recommendations/queries-on-unindexed-columns)
+- [Repeated query](/optimize/recommendations/repeated-query)
+- [`SELECT/RETURNING *`](/optimize/recommendations/select-returning)
```
Comment (nikolasburk) [inline content/700-optimize/300-recordings.mdx]: Can we find a friendlier name for this? Maybe:

```suggestion
- [`Overfetching`](/optimize/recommendations/select-returning)
```

?

File: content/700-optimize/400-recommendations/500-select-returning.mdx [hunk 3]
```diff
@@ -0,0 +1,29 @@
+---
+title: 'SELECT/RETURNING *'
+metaTitle: 'Optimize Recommendations: `SELECT/RETURNING *`'
+metaDescription: "Learn about the recommendation provided by Optimize for queries that are overfetching data."
+tocDepth: 3
+toc: true
+---
+
```
Comment (nikolasburk) [inline content/700-optimize/400-recommendations/500-select-returning.mdx:8]: I'm missing an intro sentence similar to the ones we have in "Excessive number of rows returned", "Queries on unindexed columns" and "Full table scans caused by LIKE operations"

image

image

image

File: content/700-optimize/400-recommendations/500-select-returning.mdx [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+The following query might be overfetching data in queries on the `user` model:
```
Comment (nikolasburk) [inline content/700-optimize/400-recommendations/500-select-returning.mdx]:
```suggestion
The following query might be overfetching data in queries on the `User` model:
```

**Review by nikolasburk — COMMENTED** [review]
This looks great! I've left a few small/comments and suggestions.

File: content/700-optimize/300-recordings.mdx [hunk 5]
```diff
@@ -39,6 +39,8 @@ When a recording session ends, Optimize generates recommendations such as:
 - [Excessive number of rows returned](/optimize/recommendations/excessive-number-of-rows-returned)
 - [Full table scans caused by LIKE operations](/optimize/recommendations/full-table-scans-caused-by-like-operations)
 - [Queries on unindexed columns](/optimize/recommendations/queries-on-unindexed-columns)
+- [Repeated query](/optimize/recommendations/repeated-query)
+- [`Overfetching`](/optimize/recommendations/select-returning)
```
Comment (nikolasburk) [inline content/700-optimize/300-recordings.mdx]:
```suggestion
- [Overfetching](/optimize/recommendations/select-returning)
```

File: content/700-optimize/400-recommendations/500-select-returning.mdx [hunk 6]
```diff
@@ -0,0 +1,31 @@
+---
+title: 'SELECT/RETURNING *'
```
Comment (nikolasburk) [inline content/700-optimize/400-recommendations/500-select-returning.mdx]:
```suggestion
title: 'Overfetching'
```

**Review by nikolasburk — DISMISSED** [review]

**Review by mhessdev — DISMISSED** [review]

File: content/700-optimize/400-recommendations/500-select-returning.mdx [hunk 7]
```diff
@@ -0,0 +1,31 @@
+---
+title: 'Overfetching'
```
Comment (ankur-arch) [inline content/700-optimize/400-recommendations/500-select-returning.mdx:2]:
```suggestion
title: 'Over-fetching'
```

File: content/700-optimize/400-recommendations/500-select-returning.mdx [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+metaTitle: 'Optimize Recommendations: `SELECT/RETURNING *`'
+metaDescription: "Learn about the recommendation provided by Optimize for queries that are overfetching data."
```
Comment (ankur-arch) [inline content/700-optimize/400-recommendations/500-select-returning.mdx:4]:
```suggestion
metaDescription: "Learn about the recommendation provided by Optimize for queries that are over-fetching data."
```

File: content/700-optimize/400-recommendations/500-select-returning.mdx [hunk 9]
```diff
@@ -0,0 +1,31 @@
...
+tocDepth: 3
+toc: true
+---
+
+Optimize provides recommendations to help you identify and resolve performance issues caused by overfetched data.
+
+The following query might be overfetching data in queries on the `User` model:
```
Comment (ankur-arch) [inline content/700-optimize/400-recommendations/500-select-returning.mdx:11]:
```suggestion
The following query might be over-fetching data in queries on the `User` model:
```

File: content/700-optimize/300-recordings.mdx [hunk 10]
```diff
@@ -39,6 +39,8 @@ When a recording session ends, Optimize generates recommendations such as:
 - [Excessive number of rows returned](/optimize/recommendations/excessive-number-of-rows-returned)
 - [Full table scans caused by LIKE operations](/optimize/recommendations/full-table-scans-caused-by-like-operations)
 - [Queries on unindexed columns](/optimize/recommendations/queries-on-unindexed-columns)
+- [Repeated query](/optimize/recommendations/repeated-query)
+- [Overfetching](/optimize/recommendations/select-returning)
```
Comment (ankur-arch) [inline content/700-optimize/300-recordings.mdx:43]:
```suggestion
- [Over-fetching](/optimize/recommendations/select-returning)
```

File: content/700-optimize/400-recommendations/500-select-returning.mdx [hunk 11]
```diff
@@ -0,0 +1,31 @@
+---
+title: 'Overfetching'
+metaTitle: 'Optimize Recommendations: `SELECT/RETURNING *`'
+metaDescription: "Learn about the recommendation provided by Optimize for queries that are overfetching data."
+tocDepth: 3
+toc: true
+---
+
+Optimize provides recommendations to help you identify and resolve performance issues caused by overfetched data.
```
Comment (ankur-arch) [inline content/700-optimize/400-recommendations/500-select-returning.mdx]:
```suggestion
Optimize provides recommendations to help you identify and resolve performance issues caused by over-fetched data.
```

**Review by jharrell — APPROVED** [review]

**Pull request merged**